test/pkg/integration/workspace.go

```diff
@@ -30,7 +30,7 @@ import (
 const (
 	gitpodBuiltinUserID             = "builtin-user-workspace-probe-0000000"
 	perCallTimeout                  = 5 * time.Minute
-	ParallelLunchableWorkspaceLimit = 4
+	ParallelLunchableWorkspaceLimit = 10
```

Comment:
works reliably in a large vm. Ran the workspace test suite 5 times without failure